install/sessions.csv

```diff
@@ -656,7 +656,8 @@ WGTN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO+GAL+BDS+QZSS,30s,0,linz,trimble_netr9 x4,2
 WGTN,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,linz,trimble_netr9 x4,2020-02-14T01:16:38Z,9999-01-01T00:00:00Z
 WGTT,GeoNet,GNS,LEICA CRS1000,GPS,30s,4,geonet,leica_lb2 x6,1999-12-14T00:00:00Z,2006-08-24T01:20:00Z
 WGTT,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2006-08-24T01:20:30Z,2019-03-11T01:44:11Z
-WGTT,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2019-03-11T01:44:12Z,9999-01-01T00:00:00Z
+WGTT,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2019-03-11T01:44:12Z,2024-07-15T01:21:19Z
+WGTT,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,geonet,trimble_netr9 x4,2024-07-15T01:21:20Z,9999-01-01T00:00:00Z
```

Comment:
@aleks-spes @elidana For the GNSS @ tide gauge sites will the header comment be `geonet` or `linz`? If `linz` we should change the above line to:
```
WGTT,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,linz,trimble_netr9 x4,2024-07-15T01:21:20Z,9999-01-01T00:00:00Z
``` 

Otherwise all good!

Reply:
This is an interesting question @CBurton90 ! My understanding is that we use`linz` type only for PositioNZ stations. It is usefull to think about futher deployments anyway. What do you think @elidana?

Reply:
ha, very good catch @CBurton90 !
@aleks-spes is correctly saying that `linz` is only used for PositioNZ stations so far, but my gut feeling is that we should now start to acknowledge LINZ, as those stations are now operated also on their behalf. 
I would only update the latest record though , i.e. using linz only for anything post 2024/07.

I was wondering something similar for the network code us in the networks/marks.csv file to be honest. did not think about all downstream impact of such a change though.

Reply:
@elidana, @aleks-spes, and I agreed to keep this as `geonet` for now, with discussions for updates later in the GNSS at tide gauge project